modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/builder/OAuthPlatformManagedDancerBuilder.java

```diff
@@ -47,6 +47,17 @@
    */
   OAuthPlatformManagedDancerBuilder platformUrl(String platformUrl);
 
+  /**
+   * Sets the platform API version to a specific one.
+   *
+   * @param apiVersion the platform API version
+   * @return {@code this} builder
+   * @since 4.3.1 4.4.0
+   */
+  default OAuthPlatformManagedDancerBuilder apiVersion(String apiVersion) {
```

Comment:
this method should be introduced in 4.4.0 only. 4.3.x should deal with sys prop only

Reply:
Done